conan/api/model.py

```diff
@@ -33,6 +35,70 @@ def __repr__(self):
         return str(self)
 
 
+class MultiPackagesList:
+    def __init__(self):
+        self.lists = {}
+
+    def __getitem__(self, name):
+        try:
+            return self.lists[name]
+        except KeyError:
+            raise ConanException(f"'{name}' doesn't exist is package list")
+
+    def add(self, name, pkg_list):
+        self.lists[name] = pkg_list
+
+    def add_error(self, remote_name, error):
+        self.lists[remote_name] = {"error": error}
+
+    def serialize(self):
+        return {k: v.serialize() if isinstance(v, PackagesList) else v
+                for k, v in self.lists.items()}
+
+    @staticmethod
+    def load(file):
+        content = json.loads(load(file))
+        result = {}
+        for remote, pkglist in content.items():
+            if "error" in pkglist:
+                result[remote] = pkglist
+            else:
+                result[remote] = PackagesList.deserialize(pkglist)
+        pkglist = MultiPackagesList()
+        pkglist.lists = result
+        return result
+
+    @staticmethod
+    def load_graph(graphfile, graph_recipes=None, graph_binaries=None):
+        graph = json.loads(load(graphfile))
+        pkglist = MultiPackagesList()
+        cache_list = PackagesList()
+        if graph_recipes is None and graph_binaries is None:
+            recipes = ["*"]
+            binaries = ["*"]
+        else:
+            recipes = [r.lower() for r in graph_recipes or []]
+            binaries = [b.lower() for b in graph_binaries or []]
+
+        pkglist.lists["Local Cache"] = cache_list
```

Comment:
Then, the `graphfile` param and the function itself are only supposed to fill the `Local Cache` field, aren't they? Should `load_graph` mention anything in a docstring to clarify it? If `load` are adding `remotes` and this is not, I think both docstrings could be useful.

Reply:
This is a good point.

Indeed, this is hardcoding the assumption that ``load_graph`` at the moment only happens for packages that have been installed in the cache, and then exists there for sure. 

We need to check if there is some scenario with ``graph info`` that do not retrieve the binaries to the local cache if this makes sense or not. I'll have a look.